init-tools.sh

```diff
@@ -9,7 +9,12 @@ __DOTNET_CMD="$__DOTNET_PATH/dotnet"
 if [ -z "${__BUILDTOOLS_SOURCE:-}" ]; then __BUILDTOOLS_SOURCE=https://dotnet.myget.org/F/dotnet-buildtools/api/v3/index.json; fi
 export __BUILDTOOLS_USE_CSPROJ=true
 __BUILD_TOOLS_PACKAGE_VERSION=$(cat "$__scriptpath/BuildToolsVersion.txt" | sed 's/\r$//') # remove CR if mounted repo on Windows drive
-__DOTNET_TOOLS_VERSION=$(cat "$__scriptpath/DotnetCLIVersion.txt" | sed 's/\r$//') # remove CR if mounted repo on Windows drive
+
+if [[ $2 == *"Intrinsics"* ]]; then
+    __DOTNET_TOOLS_VERSION=$(cat "$__scriptpath/DotnetCLIVersion.netcoreapp.latest.txt" | sed 's/\r$//') # remove CR if mounted repo on Windows drive
```

Comment:
(nit) there is some duplication here that can be factored out.  Create a new variable to hold the file name, setting it to different values based on the Configuration.  Then set ` __DOTNET_TOOLS_VERSION` in a single place, with the `| sed...` piece also in 1 place.